components/webhook-mgt/org.wso2.carbon.identity.webhook.metadata/src/main/java/org/wso2/carbon/identity/webhook/metadata/internal/dao/EventAdapterMetadataDAO.java

```diff
@@ -26,7 +26,7 @@
 /**
  * Data Access Object interface for managing event adopters metadata.
  */
-public interface EventAdopterMetadataDAO {
+public interface EventAdapterMetadataDAO {
 
     List<Adapter> getAdopters() throws WebhookMetadataException;
```

Comment:
I think this method name also needs to be refactored

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/7009/commits/4793bde4fe4381402954767e94e605d7b50b5c20